charts/stardog/values.yaml

```diff
@@ -108,33 +109,7 @@ zookeeper:
       memory: 2Gi
       cpu: 1
   image:
-    repository: stardog/k8szk
-    tag: 3.4.14
+    repository: bitnami/zookeeper
+    tag: 3.5.7
     pullPolicy: IfNotPresent
-  env:
-    ZK_LOG_LEVEL: INFO
-    ZK_DATA_DIR: /data
-    ZK_DATA_LOG_DIR: /data/datalog
-    ZK_LOG_DIR: /data/log
-  command:
-    - /bin/bash
-    - -xec
-    - zkGenConfig.sh && exec zkServer.sh start-foreground
-  livenessProbe:
```

Comment:
So I see the bitnami charts provide access to the [probe timings](https://github.com/bitnami/charts/blob/master/bitnami/zookeeper/values.yaml#L326). I'm wondering if it's worth setting those in our default value to highlight that this is possible. People upgrading from the previous chart versions may have tuned those already for the K8S cluster. I'm not strongly opinionated here.

Reply:
Sounds good. I added the defaults to that section so people know they are there and changeable. I commented them out so when we upgrade to newer versions of bitnami ZK charts we continue to follow their defaults (if they change) instead of having to manually track them here.